packages/gestalt/src/TableHeaderCell.js

```diff
@@ -1,20 +1,35 @@
 // @flow strict
 import React, { type Node } from 'react';
+import cx from 'classnames';
 import styles from './Table.css';
 
 type Props = {|
   children: Node,
+  shouldBeSticky?: Boolean,
   colSpan?: number,
   rowSpan?: number,
   scope?: 'col' | 'colgroup' | 'row' | 'rowgroup',
+  previousTotalWidth?: number,
 |};
 
 export default function TableHeaderCell(props: Props): Node {
-  const { children, colSpan, scope, rowSpan } = props;
+  const { children, colSpan, scope, rowSpan, shouldBeSticky, previousTotalWidth } = props;
+  const cs = cx(styles.th, shouldBeSticky && styles.columnSticky);
 
   return (
-    <th className={styles.th} scope={scope || 'col'} colSpan={colSpan} rowSpan={rowSpan}>
+    <th
+      className={cs}
+      scope={scope || 'col'}
+      colSpan={colSpan}
+      rowSpan={rowSpan}
+      style={{
+        left: shouldBeSticky ? previousTotalWidth : undefined,
+        right: shouldBeSticky ? previousTotalWidth : undefined,
+      }}
+    >
       {children}
     </th>
   );
 }
+
+TableHeaderCell.displayName = 'TableHeaderCell';
```

Comment:
I don't think we need this displayNames. If needed, 

I think we do:

TableHeaderCell.displayName = 'TableHeaderCell';

export default TableHeaderCell;

Reply:
I'm _pretty sure_ this doesn't matter, but worth testing out.